sdk/tables/azure-data-tables/tests/test_table_client_async.py

```diff
@@ -0,0 +1,540 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import unittest
+import pytest
+import platform
+
+from azure.data.tables.aio import TableServiceClient, TableClient
+from azure.data.tables._version import VERSION
+from devtools_testutils import ResourceGroupPreparer, StorageAccountPreparer
+
+from _shared.testcase import GlobalStorageAccountPreparer, TableTestCase
+
+from azure.core.exceptions import HttpResponseError
+# ------------------------------------------------------------------------------
+SERVICES = {
+    #TableServiceClient: 'table',
+    #TableClient: 'table',
+    #TableServiceClient: 'cosmos',
+    #TableClient: 'cosmos',
+}
+
+_CONNECTION_ENDPOINTS = {'table': 'TableEndpoint', 'cosmos': 'TableEndpoint'}
+
+_CONNECTION_ENDPOINTS_SECONDARY = {'table': 'TableSecondaryEndpoint', 'cosmos': 'TableSecondaryEndpoint'}
+
+class StorageTableClientTest(TableTestCase):
```

Comment:
nit: I think this could be an issue dealt with later, but would it be possible to split up the test files a little more? Maybe into ones using connection strings